x-pack/plugins/ml/public/application/components/ml_inference/add_inference_pipeline_flyout.tsx

```diff
@@ -60,6 +60,7 @@ export const AddInferencePipelineFlyout: FC<AddInferencePipelineFlyoutProps> = (
   const createPipeline = async () => {
     setFormState({ ...formState, creatingPipeline: true });
     try {
+      // @ts-expect-error pipeline._meta is defined as mandatory
```

Comment:
I suspect the correct fix for this is for ML to use `IngestPipelineConfig` rather than `IngestPipeline` in all of the places where you've had to add this comment.
I can create a follow PR to change this.